database/mysql.js

```diff
@@ -35,55 +38,38 @@ const fs = require('fs');
             console.log(err);
           } else {
             db.query(featuresTable, function (err, results) {
-              if (err) {
-                console.log(err);
-              } else {
-                db.query(interiorTable, function (err, results) {
-                  if (err) {
-                    console.log(err);
-                  } else {
-                    cb();
-                  }
-                });
-              }
+              cb();
             });
           }
         })
       }
     });
+
+
   }
 
 
   //function to load feature mock data
   const loadFeatures = (callback) => {
     // fs.writeFileSync('/Users/apple/Code/RPT11/zillowclone/database/mockfeature.json', JSON.stringify(features.createFeatures()));
-    mockFeatures.forEach(house => {
-      let sql = 'INSERT into features (type, year_built, heating, cooling, parking, lot, days_on_zillow) VALUES (?,?,?,?,?,?,?)';
-      let params = [house.type, house.year_built, house.heating, house.cooling, house.parking, house.lot, house.days_on_zillow];
-        db.query(sql, params, (err, results) => {
-          if (err) {
-            console.log(err);
-          } else {
-            console.log("mockFeatures loaded", results);
-          }
-        });
+    let sql = 'INSERT into features (type, year_built, heating, cooling, parking, lot, days_on_zillow, bedrooms, bathrooms, interiorheating, interiorcooling, appliances, kitchen, flooring, sqft) VALUES ?';
+    let params = [];
+    features.createFeatures().forEach(house => {

```

Comment:
Another nit pick... mainly about syntax style standards. On line 57, I see that you use the "house" parameter without any "()" surrounding the param, but for most of your other anonymous arrow functions you are using parens. It might help for style consistency to use the parens for all arrow functions. Again, this is a major nit pick.

Reply:
Yea, I copied this from a piece of Clifton's original code. You can tell as I don't think I would have chained the forEach like that normally.  Usually I would set it into its own variable and then called the forEach.